package.json

```diff
@@ -8,39 +8,47 @@
   },
   "dependencies": {
     "react": "^15.4.1",
-    "react-dom": "^15.4.1",
     "react-router": "^3.0.0"
   },
   "devDependencies": {
+    "archiver": "^1.0.0",
     "babel-core": "^6.2.1",
+    "babel-jest": "^22.4.1",
     "babel-loader": "^6.2.0",
+    "babel-preset-env": "^1.6.1",
     "babel-preset-es2015": "^6.1.18",
     "babel-preset-react": "^6.16.0",
     "browser-sync": "^2.11.1",
     "browser-sync-webpack-plugin": "^1.0.1",
     "copy-webpack-plugin": "^4.0.1",
     "css-loader": "^0.23.1",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-15.4": "^1.0.5",
     "eslint": "^4.0.0",
     "extract-text-webpack-plugin": "^1.0.1",
     "file-loader": "^0.8.5",
     "html-loader": "^0.4.3",
     "html-webpack-plugin": "^2.24.1",
+    "img-loader": "^2.0.1",
     "jasmine-core": "^2.6.4",
+    "jest": "^22.4.2",
     "karma": "^1.7.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-coverage": "^1.1.1",
     "karma-firefox-launcher": "^1.0.1",
     "karma-jasmine": "^1.1.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^2.0.4",
+    "on-build-webpack": "^0.1.0",
     "raw-loader": "^0.5.1",
+    "react-addons-test-utils": "^15.6.2",
+    "react-dom": "^15.6.2",
```

Comment:
This should not be a devDependency. It is crucial to production

Reply:
The PR has been updated